lib/compat/wordpress-6.3/script-loader.php

```diff
@@ -27,7 +27,7 @@
  * }
  */
 function _gutenberg_get_iframed_editor_assets() {
```

Comment:
We are changing something on the 6.3 folder which may make it easier to miss during the backport to 6.4. I guess we could remove this function from the 6.3 file and add it updated in an equivalent wordpress-6.4 file?

Reply:
Yes, though I wanted to avoid copying over the entire thing again. And this change would still work ok even in 6.3. I added the PHP back port label so it should be fine?

Reply:
It seems a little uncommon to make changes in a 6.3 folder after 6.3 is released but yes provided we don't miss the backport and it still works it is fine by me to keep it there. Feel free to merge 👍